test/2.0-test-data/not-auditable.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Run from test/smoketest
+# Output goes in $LOGDIR (/tmp by default, created if necessary) in files and export directories
+# tagged by $LOGTAG (default "no-round-two")
+
+LOGDIR=${LOGDIR:-/tmp}
+LOGTAG=${LOGTAG:-contest-not-auditable}
+
+#
+# When there exist both a contest in County A with some non-zero margin
```

Comment:
I assume this would apply even if the margins in some of the counties are zero.
```suggestion
# When a contest which spans county boundaries exists
```

Reply:
If it's zero once everything is combined, that is all I meant.